lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -133,6 +136,34 @@ def test_credit_requirements_not_eligible(self):
         )
         self.assertContains(response, "Verification Failed")
 
+    @ddt.data(
+        (CourseMode.VERIFIED, True),
+        (CourseMode.CREDIT_MODE, True),
+        (CourseMode.HONOR, False),
+        (CourseMode.AUDIT, False),
+        (CourseMode.PROFESSIONAL, False),
+        (CourseMode.NO_ID_PROFESSIONAL_MODE, False)
+    )
+    @ddt.unpack
+    def test_credit_requirements_on_progress_page(self, enrollment_mode, display_requirement):
```

Comment:
Changing `display_requirement` to `is_requirement_displayed` will make this clearer to read for future developers, as it will be obvious this is a boolean.